autoload/lightline/colorscheme/molokai.vim

```diff
@@ -0,0 +1,34 @@
+" =============================================================================
+" Filename: autoload/lightline/colorscheme/molokai.vim
+" Author: challsted
+" License: MIT License
+" =============================================================================
+"
+let s:black = [ '#232526', 233 ]
+let s:gray = [ '#808080', 244 ]
+let s:white = [ '#f8f8f2', 234 ]
+let s:cyan = [ '#66d9ef', 81 ]
+let s:green = [ '#a6e22e', 118 ]
+let s:orange = [ '#ef5939', 166 ]
+let s:pink = [ '#f92672', 161 ]
+let s:red = [ '#ff0000', 160 ]
+let s:yellow = [ '#e6db74', 229 ]
+
+let s:p = {'normal': {}, 'inactive': {}, 'insert': {}, 'replace': {}, 'visual': {}, 'tabline': {}}
+let s:p.insert.left = [ [ s:black, s:green ], [ s:cyan, s:black ] ]
+let s:p.visual.left = [ [ s:black, s:orange ], [ s:black, s:cyan ] ]
+let s:p.replace.left = [ [ s:black, s:red ], [ s:black, s:orange ] ]
+let s:p.normal.left = [ [ s:black, s:pink ], [ s:orange, s:black ] ]
```

Comment:
I would like to make the appearance of colorschemes consistent (at least for newly created colorschemes). Would you change the colours like https://github.com/itchyny/lightline.vim#wombat-with-the-patched-font, which means cyan for normal mode, green for insert mode, yellow for visual mode and red for replace mode.